### ISSUE TITLE
v-data-table with checkbox keyboard accessibility issue

### ISSUE THREAD

**Issue by MatteoNY** [vuetifyjs/vuetify#4039]
Vuetify version: 1.0.17
Vue: 2.5.16

Visit your A La Carte page and use your keyboard (space) when you get focus onto a checkbox and the checkbox event will emit "change" and update the current checkbox as "checked". Space bar again will uncheck the current checkbox. Works as expected. 

When you visit the Data Tables components page: https://vuetifyjs.com/en/components/data-tables and do the same it will not work as expected (tested in Chrome). It appears that the change event does emit but the css class "active" is not reactive or added/removed when checked/unchecked as with a click. 

When I remove the slot "headers" the top select all works as expected but not the example version for some reason. Since it works in your A La Carte version there appears to be something missing in the  code we copied from the component's example. 

Many thanks for a great set of components. I am using the Vue CLI 3. I'm hopeful for a response and getting this to work and using it on a project in the future.

Expected functionality:
Need keyboard for accessibility on v-data-table with check-boxes when used.

When checkbox has focus on the v-data-table check-boxes and space bar is used the selected checkbox should get checked or unchecked as expected. Functionality should behave same as click  on checkbox ( checked / unchecked). 

Actual Functionality (on components page):
When checkbox has focus based on tabling/blur onto it and user presses or uses space bar checkbox does NOT get checked or unchecked as expected (like when using the click event).

Best,
Matt

**Comment by KaelWD** [comment]
There's like 100 examples on that page, it'd help if you actually said which one it was. 

Seems like this is a documentation issue anyway, https://vuetifyjs.com/en/components/data-tables#example-select works fine but https://vuetifyjs.com/en/components/data-tables#example-headers doesn't because it uses `input-value` instead of `v-model`.

**Comment by MatteoNY** [comment]
KaelWD: There is only one example on the data table page with checkboxes. On the one that works on the  A La Carte page it works as expected with space bar to check and uncheck. I messed around with using custom event but the "change" event only gives you a payload of true or false so it's difficult to say from an event "this checkbox" = checked true or false. Also tried v-model and that also had a side-effect of enabling all of the checkboxes because it's multi occurring. When I visited the La Carte page I figured there was already a solution and post a issue on GitHub to see if maybe there is an updated version or a fix.  Many thanks!
